### DISCUSSION TITLE
Can Someone give a one sentence idea of what differentiates antsAI from other affine registration functions in ANTs?

### DISCUSSION THREAD

**Discussion by joshleelab** [ANTsX/ANTs#1812] [Q&A]
Can Someone give a one sentence idea of what differentiates antsAI from other affine registration functions in ANTs?

**Comment by cookpa** [comment]
antsAI is a multi-start optimizer. It rotates and / or translates the moving image and then runs registration, and keeps track of the metric value from each starting point. It outputs the transform that has the best metric score.

It's designed to quickly test a variety of starting points in order to initialize the registration. You can pass its output to `antsRegistration` with the `-r` option.

**Reply by joshleelab** [reply]
thank you!
